journeys/0xCUBE.md

```diff
@@ -0,0 +1,51 @@
+<p align="center">
+  <img src="images/Small-Cube.png" />
+</p align="center">
+
+<h1 align="center">Hi! I'm 0xCUBE! Welcome to my profile!</h1>
+<p align="center">
+ 
+ <img src="https://badges.pufler.dev/visits/0xCUB3/0xCUB3"/> 
+ <!-- <img src="https://badges.pufler.dev/years/ritik307"/> -->
+ <img src="https://badges.pufler.dev/repos/0xCUB3"/>
+ <img src="https://badges.pufler.dev/commits/yearly/0xCUB3" />
```

Comment:
images not working

![image](https://user-images.githubusercontent.com/84712013/181920493-90c68c0f-48bf-4261-87e7-9b0d6bc1e42a.png)

Reply:
This is still not working, rest is fixed.